_posts/2024-04-26-modules-101.markdown

```diff
@@ -0,0 +1,137 @@
+---
+layout: post
+title:  "Valkey modules 101"
+authors: 
+  - dmitrypol
+date: 2024-04-26 01:01:01 -0700
+categories: update
+---
+
+## What are Valkey (or Redis) modules?  
```

Comment:
I think for branding purposes, we should omit Redis completely from this post, unless specifically talking about the license changes and why the project exists.

Reply:
Completely agree on branding.  I removed all references except the ones required by SDK code.  Working to update SDK but that's going to take a bit of time.

Reply:
updated blog to use Valkey* vars.  You can see SDK changes in https://github.com/valkey-io/valkeymodule-rs/pull/8